components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react'
+import { useTranslation } from 'next-i18next'
+
+export interface CheckStatusShippingCanadaPostProps {
+  trackingNumber?: string
+}
+
+export const CheckStatusShippingCanadaPost: FC<
+  CheckStatusShippingCanadaPostProps
+> = ({ trackingNumber }) => {
+  const { t } = useTranslation(['status', 'common'])
+
+  return (
+    <>
+      <p data-testid="shipped-canada-post" className="my-5">
+        {t('shipped-canada-post.printed-and-mailing')}
+      </p>
+      <div className="my-5">
```

Comment:
`div` should be a `p` and this `p` should be condionnally rendered based on `trackingNumber` prop